flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -689,6 +695,7 @@ void restoreInternal() throws Exception {
             LOG.debug("Re-restore attempt rejected.");
             return;
         }
+        isInitializing = true;
```

Comment:
I don't see where `isInitializing` is set to `false`

Reply:
+1, It looks a little confused.

Reply:
I guess he missed some code during migration😂

Reply:
Thanks for your quick reply.
I add the `false-set` for `isInitializing`.